pkg/config/config_template.yaml

```diff
@@ -621,7 +621,49 @@ process_config:
     process: 30
     connections: 30
 #   A list of regex patterns that will exclude a process if matched.
-#   blacklist_patterns:
+  blacklist_patterns:
+  # Linux Specific patterns
+    - "^/sbin/"
+    - "^/usr/sbin/"
+    - "^/usr/bin/vi(?:m|m.basic)?$"
+    - "^sshd:"
+    - "^/bin/bash"
+    - "^/usr/bin/bash"
+    - "^/usr/lib/systemd/*"
```

Comment:
is a `*` needed ?
I see other example like `^/usr/sbin` which correctly ignores sshd (as I can see from the molecule expectations)

Reply:
Mmm yes I think the * is not needed. I'll double check. The comparison / matching happens on the first argument of the cmd line. So in the case of `sshd` it actually matches only for `^sshd:`